sw/top_darjeeling/sw/device/silicon_creator/BUILD

```diff
@@ -315,3 +315,25 @@ opentitan_functest(
         "//sw/top_darjeeling/sw/device/runtime:print",
     ],
 )
+
+cc_library(
+    name = "spi_nor_flash",
```

Comment:
should this target be defined in `sw/top_darjeeling/sw/device/silicon_creator/BUILD` rather than `sw/lib/sw/device/silicon_creator`? it is odd to see the source files not in the same package as the target definition